include/rdma/fabric.h

```diff
@@ -648,6 +648,7 @@ struct fi_param {
 	enum fi_param_type type;
 	const char *help_string;
 	const char *value;
+	const char **aliases;
```

Comment:
This is an ABI change and breaks apps.  I *strongly* prefer we not introduce an ABI change immediately before trying to release 1.7rc1.

Reply:
If I drop the change in the `fi_param` structure, would it be OK?

Reply:
For now, I would drop the last commit and just document that the new variable will override the older variable if both are set.  Or just continue to use the old variable and drop adding a new one.  I'm hesitant to spend a lot of effort on a solution for this at this time until it becomes a bigger issue.

I think we can find a way to handle aliases using the existing API.  For example, we can add new enum fi_param_type values for deprecated variables, and have the help_string or value fields reference the name of the new variable.

I would keep your implementation around, as internally, we may end up with something like this.